<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountMangement</class>
 <widget class="QWidget" name="AccountMangement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="newOrderButton">
            <property name="text">
             <string>创建新订单</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="orderHistoryButton">
            <property name="text">
             <string>历史订单</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="basicInformationButton">
            <property name="text">
             <string>账号基本信息</string>
            </property>
            <property name="name" stdset="0">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="changePasswordButton">
            <property name="text">
             <string>密码修改</string>
            </property>
            <property name="name" stdset="0">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QStackedWidget" name="stackedWidget">
             <widget class="QWidget" name="new_order_page"/>
             <widget class="QWidget" name="history_order_page">
              <widget class="QTableView" name="orderHistoryView">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>591</width>
                 <height>341</height>
                </rect>
               </property>
              </widget>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
